examples/big-bang/zarf.yaml

```diff
@@ -43,9 +43,9 @@ components:
           - "kustomizations/bigbang"
     # 1. helm template bigbang ./chart |  yq e '. | select(.kind == "GitRepository") | "- " + .spec.url + "@" + .spec.ref.tag' -
     # 2. Add the actual bigbang repo as well
-    # https://repo1.dso.mil/platform-one/big-bang/bigbang/-/tags/1.17.0
+    # https://repo1.dso.mil/platform-one/big-bang/bigbang/-/tags/1.27.1
     repos:
-      - https://repo1.dso.mil/platform-one/big-bang/bigbang.git@1.17.0
+      - https://repo1.dso.mil/platform-one/big-bang/bigbang.git@1.27.1
```

Comment:
Given the change to the bigbang version, I think it might be useful to update the repos/images we pull in to the git versions and image tags that the new bigbang version uses.

Reply:
See https://kubernetes.io/docs/tasks/access-application-cluster/list-all-running-container-images/ for a list of ways to get all the images in the cluster.